packages/vue/src/examples/pages/product/Product.vue

```diff
@@ -5,450 +5,308 @@
     SfHeading,
     SfPrice,
     SfRating,
-    SfSelect,
-    SfProductOption,
-    SfAddToCart,
+    SfIcon,
     SfTabs,
-    SfGallery,
-    SfProductCard,
-    SfCarousel,
-    SfSection,
-    SfImage,
-    SfBanner,
-    SfSticky,
+    SfProperty,
+    SfChevron,
+    SfButton,
     SfReview,
-    SfBreadcrumbs
+    SfAddToCart,
+    SfColor,
+    SfSelect,
+    SfProductOption,
+    SfBreadcrumbs,
+    SfPagination
   },
   data() {
     return {
-      qty: "1",
-      stock: 5,
-      size: "",
-      sizes: [
-        { label: "XXS", value: "xxs" },
-        { label: "XS", value: "xs" },
-        { label: "S", value: "s" },
-        { label: "M", value: "m" },
-        { label: "L", value: "l" },
-        { label: "XL", value: "xl" },
-        { label: "XXL", value: "xxl" }
-      ],
-      color: "",
-      colors: [
-        { label: "Red", value: "red", color: "#990611" },
-        { label: "Black", value: "black", color: "#000000" },
-        { label: "Yellow", value: "yellow", color: "#DCA742" },
-        { label: "Blue", value: "blue", color: "#004F97" },
-        { label: "Navy", value: "navy", color: "#656466" },
-        { label: "White", value: "white", color: "#FFFFFF" }
-      ],
-      properties: [
-        {
-          name: "Product Code",
-          value: "578902-00"
-        },
-        {
-          name: "Category",
-          value: "Pants"
-        },
-        {
-          name: "Material",
-          value: "Cotton"
-        },
-        {
-          name: "Country",
-          value: "Germany"
-        }
-      ],
-      products: [
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productA.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productB.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productC.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productA.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productB.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
-        },
-        {
-          title: "Cream Beach Bag",
-          image: "assets/storybook/Home/productC.jpg",
-          price: { regular: "50.00 $" },
-          rating: { max: 5, score: 4 },
-          isOnWishlist: false
+      selectedColor: undefined,
```

Comment:
IMO we can use the first value from the property by default or use replace to "" because yous select nothing, *is defined